src/livekit/usePreviewDevice.tsx

```diff
@@ -0,0 +1,167 @@
+import { log } from "@livekit/components-core";
```

Comment:
This is a c+p from livekit?

Reply:
yes. It is not necassary with the work around I made here: https://github.com/vector-im/element-call/pull/1119#discussion_r1232012136

The plan was to create the tracks outside of the usePreviewDevice call and than c+p the usePreviewDevice hook and refactor it so that it can take an initial track.